pages/bare_metal_cloud/dedicated_servers/partitioning_ovh/guide.en-gb.md

```diff
@@ -30,150 +30,341 @@ Providing in-depth details about partitioning can help customers understand why:
+> @api {v1} /dedicated/server GET /dedicated/server/{serviceName}/specifications/hardware
 >
 
-### Partitioning using the API
+Example reply:
 
-#### Template Concepts
+```json
+{
+  "bootMode": "uefi",
+  "defaultHardwareRaidSize": null,
+  "description": "SCALE-7 - AMD Epyc 7763",
+  "expansionCards": null,
+  "threadsPerProcessor": 128,
+  "memorySize": {
+    "unit": "MB",
+    "value": 524288
+  },
+  "defaultHardwareRaidType": null,
+  "diskGroups": [
+    {
+      "defaultHardwareRaidType": null,
+      "numberOfDisks": 2,
+      "diskGroupId": 1,
+      "diskType": "SSD",
+      "defaultHardwareRaidSize": null,
+      "diskSize": {
+        "unit": "GB",
+        "value": 480
+      },
+      "description": "2 X Disk SSD 480 GB, JBOD",
+      "raidController": null
+    },
+    {
+      "defaultHardwareRaidType": null,
+      "numberOfDisks": 2,
+      "diskGroupId": 2,
+      "diskType": "NVME",
+      "defaultHardwareRaidSize": null,
+      "diskSize": {
+        "unit": "GB",
+        "value": 1920
+      },
+      "raidController": null,
+      "description": "2 X Disk NVME 1920 GB, JBOD"
+    }
+  ],
+  "processorArchitecture": "x86_64",
+  "coresPerProcessor": 64,
+  "processorName": "Epyc7763",
+  "formFactor": "1u",
+  "motherboard": "S8036-M",
+  "numberOfProcessors": 1,
+  "usbKeys": null
+}
+```
+
+In this example, there are 2 disk groups:
 
-When triggering an OS installation, the customer can either choose between several OVHcloud templates or choose between one of their personal templates (based on an OVHcloud template).
+- the first one (diskGroupId=1) contains 2 x 480 GB disks.
+- the second one (diskGroupId=2) contains 2 x 1,9 TB disks.
+
+Example of Debian 12 (Bookworm) OS installation on the diskGroup 2:
 
 > [!api]
 >
-> @api {v1} /dedicated/server POST /dedicated/server/{serviceName}/install/start
+> @api {v1} /dedicated/server POST /dedicated/server/{serviceName}/reinstall
 >
 
-**OVHcloud templates <a name="OVHcloudtemplates"></a>**
+```json
+{
+    "operatingSystem": "debian12_64",
+    "storage": [
+        {
+            "diskGroupId": 2
+        }
+    ]
+}
+```
 
-An OVHcloud template is an official template that is nearly identical to the official software editor/vendor cloud-ready image.
+> [!primary]
+>
+> By default, the OS will be installed on diskGroupId 1.
+>
 
-To list available templates (depending on your server's hardware), use the following API call:
+<br />
 
-> [!api]
+> [!warning]
 >
-> @api {v1} /dedicated/installationTemplate GET /dedicated/installationTemplate
+> For the moment API only supports OS installation and storage customization on 1 single disk group. From 1 up to all disks of the selected disk group can be involved in the storage customization. Nevertheless, other disks of the disk group will be erased but still visible in the freshly new installed OS and can be used/configured afterward for data.
```

Comment:
```suggestion
> For the moment API only supports OS installation and storage customization on 1 single disk group. From 1 up to all disks of the selected disk group can be involved in the storage customization. Nevertheless, other disks of the disk group will be erased but  they will still be visible in the freshly new installed OS and can be used/configured afterward for data.
```

Reply:
```suggestion
> For the moment API only supports OS installation and storage customization on 1 single disk group. From 1 up to all disks of the selected disk group can be involved in the storage customization. Nevertheless, other disks of the disk group will be erased but they will still be visible in the freshly new installed OS and can be used/configured afterward for data.
```

Reply:
(same suggestion, just to fix a duplicated space)